deps/hiredis/examples/example-redismoduleapi.c

```diff
@@ -52,7 +52,7 @@ void disconnectCallback(const redisAsyncContext *c, int status) {
 /*
  * This example requires Redis 7.0 or above.
  *
- * 1- Compile this file as a shared library. Directory of "redismodule.h" must
+ * 1- Compile this file as a shared library. Directory of "valkeymodule.h" must
```

Comment:
Please don't change this. Hiredis is a vendored dependency.

```suggestion
 * 1- Compile this file as a shared library. Directory of "redismodule.h" must
```